modes/company/evil-collection-company.el

```diff
@@ -98,7 +102,8 @@ be set through custom or before evil-collection loads."
       (company-tng-configure-default)))
 
   ;; Make `company-mode' not show popup when not in supported state
-  (advice-add 'company-call-backend :before-while 'evil-collection-company-supported-p))
+  (add-hook 'evil-mode-hook 'evil-collection-company-popup))
```

Comment:
What happens if you toggle evil-mode on and off? It seems like the advice in this case is stick, e.g. won't go away.